code/modules/power/tesla/energy_ball.dm

```diff
@@ -105,9 +113,20 @@
 		qdel(Orchiectomy_target)
 
 	else if(orbiting_balls.len)
-		dissipate() //sing code has a much better system.
+		dissipate(delta_time) //sing code has a much better system.
+
+/obj/energy_ball/proc/dissipate(delta_time)
```

Comment:
Ironically this never worked before because no one actually forwarded the delta time to this proc on tesla

Reply:
delta time is for cowards anyway

Reply:
what did he mean by this